exploratory/rMorton/.Rproj.user/C5F3700/build_options

```diff
@@ -0,0 +1,7 @@
+auto_roxygenize_for_build_and_reload="0"
```

Comment:
Should this file be tracked in git? Looks like something that should be git ignored.

Reply:
It depends. If someone wanted to take over the project at where is was, this Rproj file gets them to precise place in the project. I can just gitignore though.